bases/world_oecd_public_finance/country_data/table_config.yaml

```diff
@@ -0,0 +1,2591 @@
+      is_partition:
+    - name: revenue_property_income_except_interest
+      bigquery_type: float64
+      description: Property income received (except interest receipts)
+      temporal_coverage:
+          - 1960(1)2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: expenditure_education
+      bigquery_type: float64
+      description: Education
+      temporal_coverage:
+          - 1970(1)2017
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: expenditure_health
+      bigquery_type: float64
+      description: Health
+      temporal_coverage:
+          - 1970(1)2017
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: expenditure_wage_intermediate_consumption
+      bigquery_type: float64
+      description: Wages & Interm. Consumption
+      temporal_coverage:
+          - 1986(1)2017
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: expenditure_pension
+      bigquery_type: float64
+      description: Pensions
+      temporal_coverage:
+          - 1980(1)2017
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: expenditure_sickness_disability
+      bigquery_type: float64
+      description: Sickness & Disability
+      temporal_coverage:
+          - 1980(1)2017
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: expenditure_unemployment_benefit
+      bigquery_type: float64
+      description: Unemployment Benefits
+      temporal_coverage:
+          - 1980(1)2017
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: expenditure_family_children
+      bigquery_type: float64
+      description: Family & Children
```

Comment:
colocaria na descrição Expenditure Family and Children. Sempre pense no usuário final. Eu, se pegar essa coluna, não sei do que se trata.